plugins/recorder-resources/src/screen-recorder.ts

```diff
@@ -19,21 +19,33 @@ import { TusUploader, type Uploader, type Options } from './uploader'
 export class ScreenRecorder {
   private readonly recorder: Recorder
   private readonly uploader: Uploader
+  static getMediaStream = async (options?: DisplayMediaStreamOptions): Promise<MediaStream> =>
```

Comment:
@aonnikov Do you have any thoughts on how could we solve this without static?

Reply:
have you tried to override `navigator.mediaDevices.getDisplayMedia` instead?

Reply:
nice idea, fixed ;)